src/content/chainlink-automation/tutorials/vrf-sub-monitor.mdx

```diff
@@ -1,294 +1,131 @@
 ---
 section: automation
 date: Last Modified
-title: "VRF Subscription Monitor"
+title: "VRF Subscription Balance Monitor"
 whatsnext: { "FAQs": "/chainlink-automation/faqs" }
 ---
 
-import { Aside, CodeSample } from "@components"
+import { Address, Aside, CodeSample, CopyText } from "@components"
 
-Automatically top-up your VRF subscription balances using Chainlink Automation to ensure there is sufficient funding for requests.
+Automatically top-up your VRF subscription balances using Chainlink Automation to ensure there is sufficient funding for requests. [View the code on GitHub](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/dev/VRFSubscriptionBalanceMonitor.sol).
 
-View the template [here](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/dev/VRFSubscriptionBalanceMonitor.sol).
+<Aside type="caution" title="Disclaimer">
```

Comment:
Add an objective section explaining more about the use case and purpose of the tutorial.

Reply:
Added